content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -367,7 +367,7 @@ fail validation.
 				<p><strong>Restricted Fields</strong></p>
 				<ul>
 					<li><code>spec.securityContext.runAsUser</code></li>
-					<li><code>spec.containers[*].securityContext.runAsUser</code></li>
+				<li><code>spec.containers[*].securityContext.runAsUser</code></li>
```

Comment:
Can you please revert this line back to its original state? 
The rest of the changes are good.